addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2416,14 +2416,10 @@ options.registry.Parallax = options.Class.extend({
      */
     async selectDataAttribute(previewMode, widgetValue, params) {
```

Comment:
You are asking for review again (or maybe it is a GitHub mistake?) but you did not adapt as I asked :wink: What I am asking is to not use `selectDataAttribute` anymore. You'll see that the code will get clearer.

In the xml, only use for each button:
```xml
<we-button data-set-parallax-type="..."
```

Then in JS, define:
```javascript
async setParallaxType(previewMode, widgetValue, params) {
    const speed = ...; // Something which depends on widgetValue
    this.$target[0].dataset.scrollBackgroundRation = speed;

   if (widget == aZoomValue) {
      this.$target[0].dataset.parralaxType = ...
   }
   
   // ...
}
```
}
```

Reply:
I'm sorry for that, I didn't understand what you asked. Now seems to reflect what you said.
Also let me know if using a the `_configureParallaxType` private function is good for you. I introduced it since from my point of view separate some concerns and make the code more readable.